### PR TITLE
HDDS-2201. Rename VolumeList to UserVolumeInfo.

### PR DESCRIPTION
**Pull request by anuengineer**
https://issues.apache.org/jira/browse/HDDS-2201

Under Ozone Manager, The volume points to a structure called volumeInfo, Bucket points to BucketInfo, Key points to KeyInfo. However, User points to VolumeList. duh?

This JIRA proposes to refactor the VolumeList as UserVolumeInfo. Why not, UserInfo, because that structure is already taken by the security work of Ozone Manager.

How tested: Ran unit.sh

### CODE REVIEW

File: hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/codec/UserVolumeInfoCodec.java [hunk 1]
```diff
@@ -27,22 +28,22 @@
 /**
  * Codec to encode VolumeList as byte array.
```
Comment (dineshchitlangia) [inline hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/codec/UserVolumeInfoCodec.java]: Javadoc needs to be updated

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeRequest.java [hunk 2]
```diff
@@ -53,7 +53,7 @@ public OMVolumeRequest(OMRequest omRequest) {
    * @return VolumeList - updated volume list for the user.
```
Comment (dineshchitlangia) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeRequest.java]: Javadoc needs to be updated.

File: hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/package-info.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+/**
+ * Tests for OM Response.
+ */
+package org.apache.hadoop.ozone.om.response;
```
Comment (dineshchitlangia) [inline hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/package-info.java]: New line missing at end of file.

File: hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/volume/package-info.java [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.ozone.om.response.volume;
```
Comment (dineshchitlangia) [inline hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/volume/package-info.java]: Missing javadoc and new line at EOF.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 77 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 1 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 10 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 12 | Maven dependency ordering for branch |
| -1 | mvninstall | 30 | hadoop-hdds in trunk failed. |
| -1 | mvninstall | 31 | hadoop-ozone in trunk failed. |
| -1 | compile | 18 | hadoop-hdds in trunk failed. |
| -1 | compile | 13 | hadoop-ozone in trunk failed. |
| +1 | checkstyle | 46 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 945 | branch has no errors when building and testing our client artifacts. |
| -1 | javadoc | 19 | hadoop-hdds in trunk failed. |
| -1 | javadoc | 17 | hadoop-ozone in trunk failed. |
| 0 | spotbugs | 1030 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| -1 | findbugs | 29 | hadoop-hdds in trunk failed. |
| -1 | findbugs | 16 | hadoop-ozone in trunk failed. |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 16 | Maven dependency ordering for patch |
| -1 | mvninstall | 32 | hadoop-hdds in the patch failed. |
| -1 | mvninstall | 35 | hadoop-ozone in the patch failed. |
| -1 | compile | 22 | hadoop-hdds in the patch failed. |
| -1 | compile | 15 | hadoop-ozone in the patch failed. |
| -1 | cc | 22 | hadoop-hdds in the patch failed. |
| -1 | cc | 15 | hadoop-ozone in the patch failed. |
| -1 | javac | 22 | hadoop-hdds in the patch failed. |
| -1 | javac | 15 | hadoop-ozone in the patch failed. |
| +1 | checkstyle | 52 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 797 | patch has no errors when building and testing our client artifacts. |
| -1 | javadoc | 19 | hadoop-hdds in the patch failed. |
| -1 | javadoc | 16 | hadoop-ozone in the patch failed. |
| -1 | findbugs | 29 | hadoop-hdds in the patch failed. |
| -1 | findbugs | 17 | hadoop-ozone in the patch failed. |
||| _ Other Tests _ |
| -1 | unit | 24 | hadoop-hdds in the patch failed. |
| -1 | unit | 22 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 2508 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.2 Server=19.03.2 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1566 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle cc |
| uname | Linux 63dc2a17bc4a 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 1921e94 |
| Default Java | 1.8.0_222 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/branch-mvninstall-hadoop-hdds.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/branch-mvninstall-hadoop-ozone.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/branch-compile-hadoop-hdds.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/branch-compile-hadoop-ozone.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/branch-javadoc-hadoop-hdds.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/branch-javadoc-hadoop-ozone.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/branch-findbugs-hadoop-hdds.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/branch-findbugs-hadoop-ozone.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-mvninstall-hadoop-hdds.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-mvninstall-hadoop-ozone.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-compile-hadoop-hdds.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-compile-hadoop-ozone.txt |
| cc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-compile-hadoop-hdds.txt |
| cc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-compile-hadoop-ozone.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-compile-hadoop-hdds.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-compile-hadoop-ozone.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-javadoc-hadoop-hdds.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-javadoc-hadoop-ozone.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-findbugs-hadoop-hdds.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-findbugs-hadoop-ozone.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/testReport/ |
| Max. process+thread count | 305 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/common hadoop-ozone/ozone-manager hadoop-ozone/tools U: hadoop-ozone |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/1/console |
| versions | git=2.7.4 maven=3.3.9 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/codec/UserVolumeInfoCodec.java (hunk 1 above)
Comment (anuengineer) [reply hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/codec/UserVolumeInfoCodec.java]: Thank you for catching this, fixed.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeRequest.java (hunk 2 above)
Comment (anuengineer) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeRequest.java]: Thank you for catching this, fixed.

File: hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/package-info.java (hunk 3 above)
Comment (anuengineer) [reply hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/package-info.java]: Thank you for catching this, fixed.

File: hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/volume/package-info.java (hunk 4 above)
Comment (anuengineer) [reply hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/response/volume/package-info.java]: Thank you for catching this, fixed.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 74 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | dupname | 1 | No case conflicting files found. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 10 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 31 | Maven dependency ordering for branch |
| -1 | mvninstall | 29 | hadoop-hdds in trunk failed. |
| -1 | mvninstall | 33 | hadoop-ozone in trunk failed. |
| -1 | compile | 18 | hadoop-hdds in trunk failed. |
| -1 | compile | 13 | hadoop-ozone in trunk failed. |
| +1 | checkstyle | 54 | trunk passed |
| +1 | mvnsite | 0 | trunk passed |
| +1 | shadedclient | 942 | branch has no errors when building and testing our client artifacts. |
| -1 | javadoc | 19 | hadoop-hdds in trunk failed. |
| -1 | javadoc | 16 | hadoop-ozone in trunk failed. |
| 0 | spotbugs | 1030 | Used deprecated FindBugs config; considering switching to SpotBugs. |
| -1 | findbugs | 31 | hadoop-hdds in trunk failed. |
| -1 | findbugs | 17 | hadoop-ozone in trunk failed. |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 15 | Maven dependency ordering for patch |
| -1 | mvninstall | 33 | hadoop-hdds in the patch failed. |
| -1 | mvninstall | 34 | hadoop-ozone in the patch failed. |
| -1 | compile | 21 | hadoop-hdds in the patch failed. |
| -1 | compile | 16 | hadoop-ozone in the patch failed. |
| -1 | cc | 21 | hadoop-hdds in the patch failed. |
| -1 | cc | 16 | hadoop-ozone in the patch failed. |
| -1 | javac | 21 | hadoop-hdds in the patch failed. |
| -1 | javac | 16 | hadoop-ozone in the patch failed. |
| +1 | checkstyle | 52 | the patch passed |
| +1 | mvnsite | 0 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 789 | patch has no errors when building and testing our client artifacts. |
| -1 | javadoc | 19 | hadoop-hdds in the patch failed. |
| -1 | javadoc | 17 | hadoop-ozone in the patch failed. |
| -1 | findbugs | 28 | hadoop-hdds in the patch failed. |
| -1 | findbugs | 16 | hadoop-ozone in the patch failed. |
||| _ Other Tests _ |
| -1 | unit | 24 | hadoop-hdds in the patch failed. |
| -1 | unit | 22 | hadoop-ozone in the patch failed. |
| +1 | asflicense | 29 | The patch does not generate ASF License warnings. |
| | | 2517 | |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=19.03.2 Server=19.03.2 base: https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/1566 |
| Optional Tests | dupname asflicense compile javac javadoc mvninstall mvnsite unit shadedclient findbugs checkstyle cc |
| uname | Linux 9d6dcf3d7824 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / d947ded |
| Default Java | 1.8.0_222 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/branch-mvninstall-hadoop-hdds.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/branch-mvninstall-hadoop-ozone.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/branch-compile-hadoop-hdds.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/branch-compile-hadoop-ozone.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/branch-javadoc-hadoop-hdds.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/branch-javadoc-hadoop-ozone.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/branch-findbugs-hadoop-hdds.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/branch-findbugs-hadoop-ozone.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-mvninstall-hadoop-hdds.txt |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-mvninstall-hadoop-ozone.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-compile-hadoop-hdds.txt |
| compile | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-compile-hadoop-ozone.txt |
| cc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-compile-hadoop-hdds.txt |
| cc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-compile-hadoop-ozone.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-compile-hadoop-hdds.txt |
| javac | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-compile-hadoop-ozone.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-javadoc-hadoop-hdds.txt |
| javadoc | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-javadoc-hadoop-ozone.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-findbugs-hadoop-hdds.txt |
| findbugs | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-findbugs-hadoop-ozone.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-unit-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/artifact/out/patch-unit-hadoop-ozone.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/testReport/ |
| Max. process+thread count | 341 (vs. ulimit of 5500) |
| modules | C: hadoop-ozone/common hadoop-ozone/ozone-manager hadoop-ozone/tools U: hadoop-ozone |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-1566/2/console |
| versions | git=2.7.4 maven=3.3.9 |
| Powered by | Apache Yetus 0.10.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by anuengineer** [comment]
Test failures are not related to this patch. Committed to trunk. @dineshchitlangia  Thank you for the review.

**Pull request merged**